WORKSPACE

```diff
@@ -16,6 +16,10 @@
 load("//tensorflow:workspace.bzl", "tf_workspace")
 tf_workspace()
 
+# Specify the minimum required bazel version.
+load("//tensorflow:tensorflow.bzl", "check_version")
+check_version("0.1.4")
```

Comment:
FWIW 0.1.4 will have no effect here because the bazel_version was added after 0.2.0 (so only > 0.2.1 will have an effect)